src/pages/saksbehandling/avsluttBehandling/avsluttRevurdering/AvsluttRevurdering.tsx

```diff
@@ -38,8 +28,8 @@ const AvsluttRevurdering = (props: { sakId: string; revurdering: Revurdering })
     const [avsluttRevurderingStatus, avsluttRevurderingAction] = useAsyncActionCreator(avsluttRevurdering);
 
     const { control, watch, handleSubmit } = useForm<AvsluttRevurderingFormData>({
-        defaultValues: { fritekst: null },
-        resolver: yupResolver(schema),
+        defaultValues: { fritekst: null, begrunnelse: null, brevvalgForForhåndsvarsel: null },
```

Comment:
Kan man gjøre "brevvalg" for forhåndsvarsel - fra backend fikk jeg inntrykket av at dette bare var noe man kunne gjøre etter et eventuelt forhåndsvarsel.

Reply:
I master per nå, gjør man jo et "valg", men det er ikke gjort noen API-endringer rundt dette enda nei.